### PR TITLE
Nominate iholder-redhat as an approver :+1: 

### PR DESCRIPTION
**Pull request by enp0s3**
Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

**What this PR does / why we need it**:

Nominate @iholder-redhat as an approver.

**Special notes for your reviewer**:

Community guidelines on eligibility:

https://github.com/kubevirt/community/blob/main/membership_policy.md#approver

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by maiqueb — APPROVED** [review]
Well deserved FWIW.

**Comment by stu-gott** [comment]
/approve
/hold

This looks good to me, but adding a hold so others can also weigh in.

**Comment by davidvossel** [comment]
@iholder-redhat, the quantity of your contributions to this project is immense and I certainly think you are a candidate for becoming an approver. You are a solid engineer and execute well on large scoped projects.

An approver needs more than a good execution record (which you certainly have). We also need evidence that candidates are making solid judgment calls during code reviews. This can look like steering people to simple pragmatic solutions, pointing out areas that impact api compatibility or upgrade path issues, offering alternative api suggestions that adhere closer to industry norms, and anything else that helps prove they have a depth of knowledge of the ecosystem and are able to apply that knowledge when offering direction to others.

**Can you (and anyone else that would like to be involved) help us by providing a few examples of where you think you’ve met this criteria during the code review process?** Examples could include things like the following…

* Steering people towards making better API structuring decisions that more closely align with the k8s ecosystem as a whole
* Identifying potential upgrade path issues
* Identifying potential api incompatibility issues
* Identifying inefficient use of client API calls
* Any other code review feedback that resulted in a substantially better outcome for a critical code path

I think there are a couple of paths here, both of which led to you being an approver.

1. The evidence shows you are applying and passing along a depth of ecosystem knowledge during code reviews.
2. The evidence is incomplete, but we identify the gaps that need to be filled and create an actionable plan to gather the future evidence we need to meet approver status.

I want to apologize because I think we collectively made a mistake [last time you were nominated as approver](https://github.com/kubevirt/kubevirt/pull/6528#issuecomment-937596182). We didn't capture the outcome of the discussion you and @rmohr had and it wasn't disclosed what the steps are that you two agreed upon. To fix this moving forward, I'm communicating my expectations directly on this PR so we maintain a history here.

**Pull request closed without merging**
